meshroom/nodes/aliceVision/CameraInit.py

```diff
@@ -67,7 +67,7 @@
                                  " * estimated: estimated from metadata and/or sensor width. \n"
                                  " * unknown: unknown camera parameters (can still have default value guess)\n"
                                  " * none: not set",
-                     values=("calibrated", "estimated", "unknown", "none"),
+                     values=("calibrated", "estimated", "lcp", "unknown", "none"),
```

Comment:
New `lcp` option should also be added to the attribute description

Reply:
@demoulinv ?